docs/4.5. Behaviour - Resource Servers.md

```diff
@@ -4,19 +4,26 @@ _(c) AMWA 2019, CC Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0)_
 
 ## Public keys
 
-Resource Servers SHOULD seek to fetch public keys from the Authorization Server at least once every hour, altering
-their retrieval time by a value between 0-60 seconds to avoid overloading the Authorization Server due to Resource
-Servers synchronising their retrieval time.
+Resource Servers SHOULD seek to fetch public keys from one Authorization Server in the deployment at least once every
+hour, altering their retrieval time by a value between 0-60 seconds to avoid overloading the Authorization Server due to
+Resource Servers synchronising their retrieval time.
 
-If a Resource Server is unable to contact the Authorization Server, the Resource Server MUST implement a random
-back-off mechanism to avoid overloading the Authorization Server in the event of a system restart. Also if a
-Resource Server is unable to contact an Authorization Server, the Resource Server MAY assume currently held public
-keys remain valid until it is able to re-establish a connection to an Authorization Server.
+If a Resource Server is unable to contact an Authorization Server, the Resource Server MUST implement a random back-off
+mechanism to avoid overloading the Authorization Server in the event of a system restart. If a Resource Server is unable
+to contact an Authorization Server, the Resource Server MAY assume currently held public keys remain valid until it is
+able to re-establish a connection to an Authorization Server.
```

Comment:
How does this apply to list navigation? ie multiple auth servers?

Reply:
Hopefully the next commit improves this. I'm not sure if we need a general copy/edit/paste from the IS-04 error conditions section (https://github.com/AMWA-TV/nmos-discovery-registration/blob/v1.3.x/docs/4.1.%20Behaviour%20-%20Registration.md#error-conditions) too. At present the list navigation isn't obviously covered anywhere else in the spec.

Reply:
Perhaps just a link?